crates/holochain/src/conductor/conductor/tests.rs

```diff
@@ -1172,3 +1172,123 @@ async fn test_init_concurrency() {
     assert_eq!(num_calls_clone.fetch_add(0, Ordering::SeqCst), 100);
     assert_eq!(num_inits_clone.fetch_add(0, Ordering::SeqCst), 1);
 }
+
+/// Check that an app can be installed with deferred memproof provisioning and:
+/// - all status checks return correctly while still provisioned,
+/// - no zome calls can be made while awaiting memproofs,
+/// - cells cannot be cloned while awaiting memproofs,
```

Comment:
This isn't tested yet

Reply:
Now it is (we decided in this PR it's OK to let cells be cloneable at this point)